pootle/templates/editor/units/edit.html

```diff
@@ -201,6 +201,7 @@
                 alternativeSources: {{ altsrcs|to_js }},
                 fileType: '{{ filetype }}',
                 suggestions: {{ suggestions_dict|to_js }},
+                onChange: function(values) { $('#char-count-{{ unit.id }}').html(values.map(function(e) { return e.length; }).join(' \ ')); },
```

Comment:
can we move javascript to one of the js files - probs `/static/js/editor/app.js` you can use jquery to listen for change

Reply:
also, this prevents the submit button from working